server/src/main/java/org/opensearch/node/Node.java

```diff
@@ -686,7 +686,8 @@ protected Node(
                     repositoriesServiceReference::get,
                     settings,
                     clusterService.getClusterSettings(),
-                    threadPool::preciseRelativeTimeInNanos
+                    threadPool::preciseRelativeTimeInNanos,
+                    threadPool
```

Comment:
nit: If we are passing threadpool, we can use `threadPool::preciseRelativeTimeInNanos` from the same parameter.